src/TabList/TabList.tsx

```diff
@@ -0,0 +1,23 @@
+import { TabItem } from '../TabItem';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+};
+
+export const TabList: React.FC<Props> = ({ tabs, selectedTabId }) => {
+  return (
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <TabItem
+            tab={tab}
+            selectedTabId={selectedTabId}
+            key={tab.id}
```

Comment:
Explain why you transfer the "key" but then don't use it?

Reply:
![image](https://user-images.githubusercontent.com/119679758/222655700-7dd0af5a-7ccd-4966-8d8c-8e8f7e64f2ea.png)